org.virtualxt.VirtualXT.appdata.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file should be submitted upstream.

Reply:
Yes. It will be included in the next stable release.